guides/routing/htmlforms.html

```diff
@@ -0,0 +1,246 @@
+    <pre><code class="language-swift">func getFormHandler(user: Form, respondWith: (Form?, RequestError?) -> Void) {
+    print("Codable GET route: \(user.name)")
+    if let age = user.age { print("is \(age) years old") }
+    if(user.isDeveloper) { print("and they are a developer") }
+    respondWith(user, nil)
+}</code></pre>
+    <p class="sentence">7. Restart your server and go to <a href="http://localhost:8080/formwebpage.html" target="_blank">localhost:8080/formwebpage.html</a>. Send the "Get to Codable route" form and in the Xcode logger you should see our printed message with the form data.</p>
+
+
+    <h2><a name="raw-routes" style="padding-top: 100px;"></a>Setup the Raw Routes</h2>
+    <p class="sentence">We will now create a Raw route to recieve the POST request from an HTML form.</p>
+    <p class="sentence">1. Inside "postInit", Register the Raw POST route:</p>
+    <pre><code class="language-swift">router.post("/raw") { request, response, next in
+    guard let user = try? request.read(as: Form.self)
+    else {
+        return try response.status(.unprocessableEntity).end()
+    }
+    print("Raw POST route: \(user.name)")
+    if let age = user.age { print("is \(age) years old") }
+    if(user.isDeveloper) { print("and they are a developer") }
+    response.status(.created).send(json: user)
+    next()
+}</code></pre>
+    <p class="sentence">This will function the same as the Codable POST route above, to receive and parse the HTML form.</p>
+    <p class="sentence">2. Restart your server and go to <a href="http://localhost:8080/formwebpage.html" target="_blank">localhost:8080/formwebpage.html</a>. Send the "POST to Raw route" form and in the Xcode logger you should see our printed message with the form data.</p>
+    <p class="sentence">3. After the Raw POST route, Register the Raw GET route:</p>
+    <pre><code class="language-swift">router.get("/raw") { request, response, next in
+    guard let user = request.getQueryParameters(as: Form.self)
+    else {
+        return try response.status(.unprocessableEntity).end()
+    }
+    print("Raw GET route: \(user.name)")
+    if let age = user.age { print("is \(age) years old") }
+    if(user.isDeveloper) { print("and they are a developer") }
+    response.status(.created).send(json: user)
+    next()
+}</code></pre>
+    <p class="sentence">4. Restart your server and go to <a href="http://localhost:8080/formwebpage.html" target="_blank">localhost:8080/formwebpage.html</a>. Send the "GET to Raw route" form and in the Xcode logger you should see our printed message with the form data.</p>
+    <p class="sentence">This will function the same as the Codable GET route above, to receive and parse the HTML form.</p>
+    <p class="sentence">Congratulations!! You have just created and hosted a HTML forms webpage. Then set up a server to receive and parse these forms using four different methods! </p>
```

Comment:
a HTML forms -> an HTML forms
I don't think the sentence break works here I think it'd be better as "You have just created and hosted an HTML forms webpage and set up a server to receive and parse these forms using four different methods!"